code/modules/paperwork/paper_biscuit.dm

```diff
@@ -0,0 +1,80 @@
+/obj/item/folder/biscuit
+	name = "A biscuit card"
+	desc = "An biscuit card"
+	icon_state = "paperbiscuit"
+	bg_color = "#b88f3d"
+	var/cracked = FALSE //Is biscuit cracked open or not?
+
+/obj/item/folder/biscuit/suicide_act(mob/living/user)
+	user.visible_message(span_suicide("[user] tries to eat the 'biscuit'! [user.p_theyre()] trying to commit suicide!"))
+	return TOXLOSS
+
+/obj/item/folder/biscuit/update_overlays()
+	. = ..()
+	if(contents.len && cracked) //Shows overlay only when it has content and is cracked open
+		. += "paperbiscuit_paper"
+		. -= "folder_paper"
+
+/obj/item/folder/biscuit/examine()
+	. = ..()
+	if(length(contents))
+		. += span_notice("To reach contents you need to crack it open.")
+
+/obj/item/folder/biscuit/proc/crack_check(mob/user)
+    if (cracked)
+        return TRUE
+    balloon_alert(user, "need to crack it open first!")
```

Comment:
```suggestion
    balloon_alert(user, "unopened!")
```

balloon_alerts should be more brief, this carries the point imo, but you can do whatever you want